server/svix-server/src/main.rs

```diff
@@ -90,19 +94,58 @@ async fn main() {
         );
     }
 
+    let otel_layer = cfg.opentelemetry_address.as_ref().map(|addr| {
```

Comment:
Shouldn't otel_layer be optional and only added if needed?

Reply:
`otel_layer` here is an `Option` as is the `opentelemetry_address`.

Reply:
Oh, adding layers supports options? (the `with()` below)

Reply:
Yup!